assets/sass/styles.scss

```diff
@@ -56,7 +56,38 @@ body {
     max-width: 800px;
     overflow-y: auto;
 	background-color: #2a2822;
-	padding: 30px 30px;
+	padding: 0;
 	box-sizing: border-box;
     box-shadow: 0 0 30px rgba(0, 0, 0, 0.3), 0 0 10px rgba(0, 0, 0, 0.4);
+
+	&::-webkit-scrollbar {
```

Comment:
Seems like webkit-scrollbar is chromium only. just pointing it out. i believe firefox can use the property `scrollbar-color` on the container to change colour.

https://stackoverflow.com/a/54101063
https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar